format-translators-module/format-translators-module/parser/structure_to_triples_parser.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "parser/types.hpp"
+
+#include <sc-memory/sc_memory.hpp>
+
+namespace formatTranslators
+{
+class StructureToTriplesParser
+{
+public:
+  void ParseStructure(
+      ScMemoryContext * memoryContext,
+      ScAddr const & structureToTranslateAddr,
+      ScAddrToValueUnorderedMap<size_t> const & keyElementsOrderMap,
+      ScAddr const & identifiersLanguageAddr,
+      Triples & structureTriplesMap);
+
+private:
+  ScMemoryContext * context = nullptr;
+  ScAddr structureToTranslate;
+  ScAddr identifiersLanguage;
+  ScAddrToValueUnorderedMap<size_t> keyElementsOrder;
```

Comment:
Don't use size_t in container. It is platform dependent type. Use uint32_t or uint64_t instead